### DISCUSSION TITLE
[2.x] 布局的API labelWidth 和 labelStyle 的一点建议

### DISCUSSION THREAD

**Discussion by 708801556** [alibaba/formily#1235] [Ideas]
labelWidth ：建议可以自己添加单位，而不是默认是px，这样更灵活。比如我想设置5个中文的时候直接 5em。这样改变字体大小的时候，不用重新设置宽度也能自适应。

labelStyle：建议支持字符串形式css。CSSProperties 写起来麻烦。

**Comment by janryWang** [comment]
labelStyle这个是走的标准React style，如果支持字符串形式，就不标准了，不过你说的width可以支持字符串这个可以支持
